src/AMD-newlib.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="AMD-newlib"
+   name="AMD newlib License" listVersionAdded="">
```

Comment:
Can you add to listVersionAdded=3.24, please?

Reply:
Ack, done.